app/config/message.js

```diff
@@ -23,8 +23,10 @@ const mqSchema = Joi.object({
     topic: Joi.string(),
     type: Joi.string().default('subscription')
   },
-  statementTopic: {
-    address: Joi.string()
+  statementSubscription: {
+    address: Joi.string(),
+    topic: Joi.string(),
+    type: Joi.string().default('subscription')
```

Comment:
As above, need to keep the original.  Maybe consider something about data in the name so it's clear is the feed of customer/calculation data